samples/caf_sensor_manager/sample.yaml

```diff
@@ -13,11 +13,11 @@ common:
     type: multi_line
     ordered: true
     regex:
-     - "sensor_sim_ctrl state:READY"
-     - "accel_sim_xyz state:ACTIVE"
+     - "main state:READY"
      - "Send sensor buffer desc address:"
      - "sensor_data_aggregator_release_buffer_event"
 
 tests:
   sample.caf_sensor_manager:
     build_only: false
+    extra_args: CONF_FILE=prj_logs.conf
```

Comment:
we have - I am just looking what will be the simplest to use

on the desk we don't need the configuration without logs - so the default one should build with logs
nrf_desktop has release build types due to being a reference design - code could go to the production device.

here we just need to get rid logs+serial in the CI. if we create the second configuration, call it _release (just like in case of desktop) and set options expected from release config (no logs, debugs, asserts, etc). It will have more sense and will be in line with what we have in other places. But if we only need to set few options I think it might really be better to set them in sample.yml